README.md

```diff
@@ -56,7 +56,7 @@ Python 3.6
     ```
     git clone https://github.com/DEX-Company/barge.git
     cd barge
-    git checkout tags/dex-2019-03-05
+    git checkout tags/dex-2019-04-10
     ./start_ocean.sh --no-brizo --no-pleuston --local-spree-node
     ```
 
```

Comment:
NOTE: you need to change the line to say test not tests:
`python3 setup.py test`

Reply:
I've added this as another command to run the complete tests in the README.md, since both

`python3 -m pytest tests`
or
`python3 setup.py test`

will run the complete unit and integration tests